subsys/bluetooth/shell/gatt.c

```diff
@@ -946,43 +950,79 @@ static int cmd_unregister_test_svc(const struct shell *sh,
 	return 0;
 }
 
-static void notify_cb(struct bt_conn *conn, void *user_data)
+static uint8_t found_attr(const struct bt_gatt_attr *attr, uint16_t handle,
+			  void *user_data)
 {
-	const struct shell *sh = user_data;
+	const struct bt_gatt_attr **found = user_data;
 
-	shell_print(sh, "Nofication sent to conn %p", conn);
+	*found = attr;
+
+	return BT_GATT_ITER_STOP;
+}
+
+static const struct bt_gatt_attr *find_attr(uint16_t handle)
+{
+	const struct bt_gatt_attr *attr = NULL;
+
+	bt_gatt_foreach_attr(handle, handle, found_attr, &attr);
+
+	return attr;
 }
 
 static int cmd_notify(const struct shell *sh, size_t argc, char *argv[])
 {
-	struct bt_gatt_notify_params params;
-	uint8_t data = 0;
+	const struct bt_gatt_attr *attr;
+	int err;
+	size_t data_len;
+	unsigned long handle;
+	static char data[CHAR_SIZE_MAX];
```

Comment:
I am wondering if we could reuse the `gatt_write_buf`, as you cannot (currently anyhow) perform both the write and notify command in parallel, right?